rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/BaseLanguageServer.java

```diff
@@ -285,5 +287,10 @@ public void connect(LanguageClient client) {
             lspDocumentService.connect(this.client);
             lspWorkspaceService.connect(this.client);
         }
+
+        @Override
+        public void registerVFS(VFSRegister registration) {
+            VSCodeVFSClient.buildAndRegister(registration.getPort());
+        }
```

Comment:
All changes in this file except this addition looks like autoformatting gone wrong